### PR TITLE
Add support to translate echoes + finalize new flashlight module

### PR DESCRIPTION
**Pull request by Tiagoquix**
This has been translated in first person and in singular.

### CODE REVIEW

File: dev/presets/package.sh [hunk 1]
```diff
@@ -27,9 +27,9 @@ for F in ../../config/cfg/presets/*; do
         printf "exec comfig/addons_setup.cfg;"
         printf "exec comfig/addons.cfg;"
         printf "modules_c;"
-        printf "exec comfig/echo.cfg;"
         printf "run_modules;"
         printf "exec comfig/addons.cfg;"
+        printf "echo_lang;"
```
Comment (mastercoms) [inline dev/presets/package.sh]: Move this back

**Review by mastercoms — CHANGES_REQUESTED** [review]

**Comment by Tiagoquix** [comment]
Done @mastercoms

**Comment by Tiagoquix** [comment]
From my testings using ``preset_low;run_modules;r_flashlightdepthtexture 1;r_flashlightrender 1;r_flashlightmodels 1;r_flashlightculldepth 1;r_flashlightscissor 1;mat_supportflashlight 1``, to enable flashlight it only requires ``shadows_very_low`` and nothing more. Not tested on Very Low, but the flashlight should work too.

``mat_supportflashlight 1`` is necessary because the game doesn't set it automatically (it only sets when the game launches).

I'll add new commits soon.

**Comment by Tiagoquix** [comment]
The flashlight module has been finalized.

@mastercoms This pull request is now ready for review.

**Review by mastercoms — APPROVED** [review]

**Pull request merged**